backend/handlers/bets/placebethandler.go

```diff
@@ -13,23 +13,22 @@ import (
 )
 
 func PlaceBetHandler(loadEconConfig setup.EconConfigLoader) func(http.ResponseWriter, *http.Request) {
+
 	return func(w http.ResponseWriter, r *http.Request) {
-		// Only allow POST requests
+
 		if r.Method != http.MethodPost {
 			http.Error(w, "Method not supported", http.StatusMethodNotAllowed)
 			return
 		}
```

Comment:
I think we agreed that if the router is enforcing methods we can remove the method check here. This one is enforced on L74 in the server file.